l10n_br_sale/README.rst

```diff
@@ -1,81 +1,98 @@
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fl10n--brazil-lightgray.png?logo=github
+    :target: https://github.com/OCA/l10n-brazil/tree/10.0/l10n_br_sale
+    :alt: OCA/l10n-brazil
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/l10n-brazil-10-0/l10n-brazil-10-0-l10n_br_sale
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/124/10.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module was written to extend the functionality of Sale to support
+and allow you to in Brazil.
+
+**Table of contents**
+
+.. contents::
+   :local:
 
 Installation
 ============
 
-To install this module, you need to:
+This module depends on:
 
-* do this ...
+* account_fiscal_position_rule_sale
+* l10n_br_account
 
 Configuration
 =============
 
-To configure this module, you need to:
-
-* go to ...
-
-Usage
-=====
-
-To use this module, you need to:
-
-* go to ...
-
-.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
-   :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/{repo_id}/{branch}
-
-
-.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
-.. branch is "8.0" for example
-
-Known issues / Roadmap
-======================
-
-TODO
+No configuration required.
 
 Bug Tracker
 ===========
 
 Bugs are tracked on `GitHub Issues <https://github.com/OCA/l10n-brazil/issues>`_.
 In case of trouble, please check there if your issue has already been reported.
-If you spotted it first, help us smashing it by providing a detailed and welcomed feedback `here <https://github.com/OCA/
-l10n-brazil/issues/new?body=module:%20
-l10n_br_account_product%0Aversion:%20
-8.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/l10n-brazil/issues/new?body=module:%20l10n_br_sale%0Aversion:%2010.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
 
+Do not contact contributors directly about support or help with technical issues.
 
 Credits
 =======
 
+Authors
+~~~~~~~
+
+* Akretion
+
 Contributors
-------------
+~~~~~~~~~~~~
 
 * Renato Lima <renato.lima@akretion.com.br>
 * Raphaël Valyi <raphael.valyi@akretion.com.br>
-* Luis Felipe Mileo <mileo@kmee.com.br> 
-* Michell Stuttgart <michell.stuttgart@kmee.com.br> 
+* Luis Felipe Mileo <mileo@kmee.com.br>
+* Michell Stuttgart <michell.stuttgart@kmee.com.br>
+
+Other credits
+~~~~~~~~~~~~~
+
+The development of this module has been financially supported by:
+
+* AKRETION LTDA - www.akretion.com
+* KMEE INFORMATICA LTDA - www.kmee.com.br
```

Comment:
honnestly although we appreciate the help from KMEE, it didn't do more than 5% of the work in this module specifically, so be mentioned as a module sponsor is a bit abusive I think...
Reciprocally, Akretion doesn't try to appear as a module co-author when our role was minor in some module...